src/extension.ts

```diff
@@ -1460,7 +1465,82 @@ async function execDraftCreateApp(appName : string, pack? : string) : Promise<vo
     }
 }
 
-function draftCreateResult(sr : ShellResult, hadPack : boolean) {
+async function execDraftCreateDebug() {
+
+    const pack = "github.com/radu-matei/draft-packs/packs/node-debug/"
+
+    if (vscode.workspace.rootPath === undefined) {
+        vscode.window.showErrorMessage('This command requires an open folder.');
+        return;
+    }
+    if (draft.isFolderMapped(vscode.workspace.rootPath)) {
+        vscode.window.showInformationMessage('This folder is already configured for draft. Run draft up to deploy.');
+        return;
+    }
+    if (!(await draft.checkPresent())) {
+        return;
+    }
+    const proposedAppName = path.basename(vscode.workspace.rootPath);
+    const appName = await vscode.window.showInputBox({ value: proposedAppName, prompt: "Choose a name for the Helm release" });
+    if (appName) {
+        await execDraftCreateApp(appName, pack);
+    }
+}
+
+async function execDraftUpDebug() {
+    const draftPath = await draft.path();
+    const term = vscode.window.createTerminal('draft logs', `bash`, ['-c', `${draftPath} logs ; bash`]);
+    term.show(true);
+
+    vscode.window.showInformationMessage("Starting draft up")
+    await draft.invokeAsync('up').then((result) => {
```

Comment:
Mixing of `await` and thenable-callback style looks odd to me.  (We have a few places where we do it with `kubectl`, but those are because the code is undergoing a long slow transition from thenables/callbacks to `async/await`.  I'd prefer not to propagate it further, unless of course there's good reason!)